dangerfile.ts

```diff
@@ -0,0 +1,261 @@
+// inspire by reacts dangerfile
+// danger has to be the first thing required!
+import { danger, markdown } from 'danger';
+import { exec } from 'child_process';
+import { loadComparison } from './scripts/sizeSnapshot';
+import replaceUrl from './packages/api-docs-builder/utils/replaceUrl';
+
+const circleCIBuildNumber = process.env.CIRCLE_BUILD_NUM;
+const circleCIBuildUrl = `https://app.circleci.com/pipelines/github/mui/material-ui/jobs/${circleCIBuildNumber}`;
```

Comment:
```suggestion
const circleCIBuildUrl = `https://app.circleci.com/pipelines/github/mui/pigment-css/jobs/${circleCIBuildNumber}`;
```

(assuming the CI is set up)

Reply:
I will change this, the CI will come in a follow up PR.